### DISCUSSION TITLE
[Bug] [Suggestion] Sending a friend request doesn't notify the other party

### DISCUSSION THREAD

**Discussion by Soulsderfuchs** [RetroAchievements/RAWeb#810] [Ideas]
When you go to send a friend request to someone in the site it actually doesn't notify them of this and of top of that they have to manually go to your profile to accept the friend request, so it would be nice to implement some kind of prompt or unique page to look at all the requests received.

**Comment by Jamiras** [comment]
They do receive an email notification (just not an on-site notification). The "unique page" was added via #1042.

**Comment by Jamiras** [comment]
Bug filed as #531
